homework/src/index.js

```diff
@@ -30,17 +31,88 @@
     return elem;
   }
 
-  function main(url) {
-    fetchJSON(url, (err, data) => {
-      const root = document.getElementById('root');
-      if (err) {
-        createAndAppend('div', root, { text: err.message, class: 'alert-error' });
-      } else {
-        createAndAppend('pre', root, { text: JSON.stringify(data, null, 2) });
-      }
+  function renderRepositories(index, repositoriesdata, container) {
+    const repoInfo = createAndAppend('div', container, { 'class': 'left-div box' });
+    const table = createAndAppend('table', repoInfo);
+
+    const repoTr = createAndAppend('tr', table);
+    createAndAppend('td', repoTr, { 'text': 'Repository :', 'class': 'label' });
+    const repoName = createAndAppend('td', repoTr, { 'id': 'repo-name' });
+    const url = createAndAppend('a', repoName, { 'target': '_blank', });
+
+    const descriptionTr = createAndAppend('tr', table);
+    const descriptionName = createAndAppend('td', descriptionTr, { 'text': 'Description :', 'class': 'label' });
+    const descriptionDetails = createAndAppend('td', descriptionTr);
+
+    const forkTr = createAndAppend('tr', table);
+    createAndAppend('td', forkTr, { 'text': 'Forks :', 'class': 'label' });
+    const forkNum = createAndAppend('td', forkTr);
+
+    const updateTr = createAndAppend('tr', table);
+    createAndAppend('td', updateTr, { 'text': 'Updated :', 'class': 'label' });
+    const updateNum = createAndAppend('td', updateTr);
+
+    url.innerText = repositoriesdata[index].name;
+    url.setAttribute('href', repositoriesdata[index].html_url);
+    if (repositoriesdata[index].description === null) {
+      descriptionName.innerHTML = '';
+      descriptionDetails.innerText = '';
+    } else {
+      descriptionName.innerText = "Description: ";
+      descriptionDetails.innerText = repositoriesdata[index].description;
+    }
+    forkNum.innerText = repositoriesdata[index].forks;
```

Comment:
You could directly set the text through the `options` parameter in the `createAndAppend()` call in  line 49.